src/js/states/admin/edit/edit.controller.js

```diff
@@ -68,6 +68,30 @@
 				} );
 			};
 
+			$scope.deleteProfile = function () {
+				if(confirm("Are you sure you want to delete this profile?")){
```

Comment:
This should be handled through a Bootstrap Modal. See error handler below for an example and documentation at http://angular-ui.github.io/bootstrap/versioned-docs/1.3.3/#/modal